### PR TITLE
editorial/copy pass

### PR DESCRIPTION
**Pull request by mking18**

### CODE REVIEW

File: docs/patterns/status-notifications/Readme.md [hunk 1]
```diff
@@ -17,15 +17,13 @@
 
 ## Overview
 
-Status Notifications will not interrupt a user from a task they are engaged in. They are provided on user action rather than as part of the page. 
-These non-dialog, event based notifications differ from Update Notifications as they are contained in bespoke UI expressions which do not update live, existing inline sections of a page.
-They provide information which will help users make a decision, communicate statuses, or provide feedback about selections.
-These notifications may open or be added to locations unrelated to the action which caused the notification to trigger.
-Additionally, they may open based on conditions a user has created or criterium they have met.
+Status Notifications inform users that something on the page has changed. These responses help users understand page processes, actions they have competed, additional options available, or actions still needing attention. Common examples include success and warning messages.
```
Comment (rebecca-larue) [inline docs/patterns/status-notifications/Readme.md:20]: "actions they have completed"

File: docs/patterns/status-notifications/Readme.md [hunk 2]
```diff
@@ -17,15 +17,13 @@
...
+
+Based on the information being communicated, status notifications can be persistent or transient. While it’s important to keep users informed when there is an update or status change, these notifications should not be page blocking.
```
Comment (rebecca-larue) [inline docs/patterns/status-notifications/Readme.md:22]: "Depending on the information..."

File: docs/patterns/status-notifications/Readme.md [hunk 3]
```diff
@@ -165,7 +160,7 @@ Persistent Status Notifications are 'in page messages', often [validation summar
 - **Must**
   - Ensure the notification will not be removed if keyboard focus or mouse hover is within or over the notification.
```
Comment (rebecca-larue) [inline docs/patterns/status-notifications/Readme.md:161]: I believe these should not have a period

File: docs/patterns/status-notifications/Readme.md [hunk 4]
```diff
@@ -165,7 +160,7 @@ Persistent Status Notifications are 'in page messages', often [validation summar
...
   - Return user focus to a logical location.
```
Comment (rebecca-larue) [inline docs/patterns/status-notifications/Readme.md:162]: No period

File: docs/patterns/status-notifications/Readme.md [hunk 5]
```diff
@@ -280,79 +279,77 @@ Persistent Status Notifications are 'in page messages', often [validation summar
 
 ### Use When
 - As confirmation that a task or process initiated by the user was completed successfully
```
Comment (rebecca-larue) [inline docs/patterns/status-notifications/Readme.md:281]: "Confirming that a task or process..."

File: docs/patterns/status-notifications/Readme.md [hunk 6]
```diff
@@ -280,79 +279,77 @@ Persistent Status Notifications are 'in page messages', often [validation summar
...
+- The message contains a rich UI experience (see [Modal](../../components/modal/))
+- The user makes a selection that does not change or add content to the page
 
-**note: This section is provided as a set of requirements used for the construction of cedar components and for projects needing custom solutions not using provided cedar components**
+### Anatomy of a Transient Status Notification
 
 <cdr-img :src="$withBase('/notifications/transientStatusAnatomy.png')" alt="Diagram for transient status notifications, annotating the required layout of the elements listed below" />
 
-Transient Status notifications have the following requirements in addition to the requirements for Persistent Status Notifications (TODO should requirements that are pertinent be additive or all inclusive?)
+Transient Status Motifications have the following requirements in addition to the requirements for Persistent Status Notifications (TODO should requirements that are pertinent be additive or all inclusive?)
```
Comment (rebecca-larue) [inline docs/patterns/status-notifications/Readme.md:301]: "Notifications"

**Review by rebecca-larue — CHANGES_REQUESTED** [review]
Just a few small changes

**Review by mhewson — APPROVED** [review]

**Pull request merged**